python/lsst/cp/pipe/ptc/cpPtcExtract.py

```diff
@@ -400,7 +400,7 @@ def run(self, inputExp, inputDims, taskMetadata, inputPhotodiodeData=None):
             sequentially by their exposure id.
         inputDims : `list`
             List of exposure IDs.
-        taskMetadata : `list` [`lsst.pipe.base.TaskMetadata`]
+        taskMetadata : `list` [`lsst.pipe.base.TaskMetadata`], optional
             List of exposures metadata from ISR.
```

Comment:
This should be commented that this is unused even if specified.  Also, below there should be a check if somebody hands it in and do a deprecation warning if so.  (And also tagged with the same TODO as above).